examples/pytorch_surrogate_model/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _examples-ml-surrogate:
+
+9 stage laser-plasma accelerator surrogate example
+==================================================
+
+Nine stages of laser-plasma accelerators.
+The laser-plasma accelerator elements are neural network surrogate models,
+previously trained and included in ``models``.
+The neural networks require normalized input data, the normalizations can be found in ``datasets``.
```

Comment:
Let's add the `.pt` repos in a Zenodo archive and download them on the fly. That way, we keep the `git` repo small :)